model_development_pipeline.py

```diff
@@ -1,26 +1,32 @@
-#%%
+# %%
 from model_development.dataset_management import download_dataset, aggregate_all_classes, construct_image_classification_dataset
 from model_development.sharktrack_trainer import SharkTrackTrainer
+from model_development.train_image_classifier import main as train_image_classifier
 from pathlib import Path
 
-#%%
+# %%
 # Download dataset
 workspace = "msc-fish-seychelles"
 project = "seychelles"
 version = 35
 object_detection_yaml = download_dataset(workspace, project, version, "yolov8")
 
-#%%
+# %%
 # Construct Image Classification Dataset
-construct_image_classification_dataset(object_detection_yaml)
+image_classification_folder = construct_image_classification_dataset(
```

Comment:
I think currently construct_image_classification_dataset does not return the folder. Could you update the code to make it return?

Reply:
@Sonnyburni